libraries/core_libs/network/include/network/tarcap/packets_handlers/common/packet_handler.hpp

```diff
@@ -39,6 +39,9 @@ class PacketHandler {
    */
   void processPacket(const PacketData& packet_data);
 
+  void requestPbftNextVotesAtPeriodRound(dev::p2p::NodeID const& peerID, uint64_t pbft_period, uint64_t pbft_round,
```

Comment:
const first

Reply:
why is this function declared in PacketHandler, when it is used only in VotesPacketHandler ?

Reply:
It is also used in `validateStandardVote` when we are requesting votes. I think it was just missed after merge.

Reply:
then it should be in ExtVotesPacketHandler

Reply:
It is used in `StatusPacketHandler : ExtSyncingPacketHandler : PacketHandler` and in `ExtVotesPacketHandler : PacketHandler`. So it couldn't only be in `ExtVotesPacketHandler`